CMakeLists.txt

```
@@ -26,14 +34,14 @@ if(BUILD_TESTING)
     block()
         set(INSTALL_GTEST OFF) # Disable GoogleTest installation
         set(BUILD_TESTING OFF) # Disable GoogleTest tests
+        set(BUILD_GMOCK OFF)
+        set(CMAKE_CXX_FLAGS) # FIXME: Do not use our project settings!
```

```suggestion
        set(CMAKE_CXX_FLAGS) # Prevent propagating our project settings
```
I don't think `FIXME` is appropriate here, feel free to reject this. This is a nitpick.

---

I do not agree with you:
```
bash-5.2$ cd build/gcc-debug/
bash-5.2$ ninja clean gtest
[1/1] Cleaning all built files...
Cleaning... 9 files.
bash-5.2$ ninja -nv gtest
[1/2] /usr/bin/g++  -I/Users/clausklein/Workspace/cpp/beman-project/exemplar/build/gcc-debug/_deps/googletest-src/googletest/include -I/Users/clausklein/Workspace/cpp/beman-project/exemplar/build/gcc-debug/_deps/googletest-src/googletest -fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=undefined -g -std=c++17 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.1.sdk -mmacosx-version-min=14.7  -DGTEST_HAS_PTHREAD=1 -MD -MT _deps/googletest-build/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o -MF _deps/googletest-build/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o.d -o _deps/googletest-build/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o -c /Users/clausklein/Workspace/cpp/beman-project/exemplar/build/gcc-debug/_deps/googletest-src/googletest/src/gtest-all.cc
[2/2] : && /usr/local/bin/cmake -E rm -f lib/libgtest.a && /usr/bin/ar qc lib/libgtest.a  _deps/googletest-build/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o && /usr/bin/ranlib lib/libgtest.a && /usr/local/bin/cmake -E touch lib/libgtest.a && :
bash-5.2$ 
```

---

okay cool!